src/main.rs

```diff
@@ -54,6 +55,11 @@ fn main() {
         println!();
     }
 
+    let config_contents = fs::read_to_string("./config_test.toml").unwrap();
+    let value = config_contents.parse::<Value>().unwrap();
+    println!("{:?}", value);
```

Comment:
Using the toml parser in this commit, the output of this showed a table containing an array, containing two strings.. so... seemed to work. But I haven't tried inserting all the code necessary to parse that type of data structure.